cmd/codegen/app/generate_clientset.go

```diff
@@ -109,11 +109,18 @@ func (o *ClientSetGenerationOptions) Run() error {
 		return errors.Wrapf(err, "ensure GOPATH is set correctly")
 	}
 
-	err = generator.InstallCodeGenerators(o.GeneratorVersion)
+	gopath := util.GoPath()
+	if !o.NotIsolated {
```

Comment:
I find this type of if statements hard to process. See above.

Reply:
So per default, you don't want to install into the GOPATH?

Reply:
exactly, it doesn't work at all. I'm not entirely sure how it works for you.